### PR TITLE
feat(exporter): exporting values from results table

### PR DESCRIPTION
**Pull request by Lockdauun**
## Description of changes
Allowing the results table to be exported regarding whats shown on the GUI

Issue number: #4 
After query:
image

Export:
image

Currently exporting table from results from GUI, llm response is not being parsed correctly into results table, therefore is exporting as above

### CODE REVIEW

File: modules/exporter/exporter.py [hunk 1]
```diff
@@ -4,4 +4,6 @@ def exportToExcel(outputFileName: str, dataframe: DataFrame) -> None:
     """
     Exports a DataFrame to an Excel file
     """
-    dataframe.to_excel(outputFileName, index=False)
\ No newline at end of file
+    print("Starting Export")
```
Comment (Semuca) [inline modules/exporter/exporter.py]: Please remove these debug statements

File: modules/exporter/journal_parser.py [hunk 2]
```diff
@@ -5,4 +5,23 @@ def journalResponsesToDataFrame(journals: Any) -> DataFrame:
     """
     Convert a list of journals with responses to a pandas DataFrame
     """
-    pass
\ No newline at end of file
+    data = []
+    row_count = journals.ui.resultsListTableWidget.rowCount()
+    print(row_count)
```
Comment (Semuca) [inline modules/exporter/journal_parser.py]: Same as above

**Review by Semuca — CHANGES_REQUESTED** [review]
Couple comments made- can you resolve merge conflicts too?

**Comment by Lockdauun** [comment]
> Couple comments made- can you resolve merge conflicts too?

Removed debug statements and solved merge conflicts

**Review by Semuca — APPROVED** [review]

**Pull request merged**